src/main/java/no/digipost/api/client/ApiService.java

```diff
@@ -175,4 +177,18 @@
      * @param avsenderenhet underenhet for et organisasjonsnummer.
      */
     SenderInformation getSenderInformation(String orgnr, String avsenderenhet);
+
+    Inbox getInbox(SenderId senderId, int offset, int limit);
```

Comment:
Ah, ok. Brukere av biblioteket forholder seg vel kanskje ikke til denne klassen. Da er det ikke så nøye for min del.